manual/src/cmds/intf-c.etex

```diff
@@ -1753,9 +1753,8 @@ gracefully, which equals the following:
 \begin{itemize}
```

Comment:
While you're here, please fix "4.05" to "4.06" on line 1751 (GitHub doesn't let me comment there).

Reply:
Done.